_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -331,6 +332,46 @@ $buttons-elevated-shadow: 3px !default;
 $buttons-border-radius: 3px !default;
 $buttons-font-weight: font-weight(semi-bold) !default;
 
+// forms
+$label-color: $text-base-color;
+$label-color-active: palette(grayscale, x-dark);
+
+$field-default-background: palette(grayscale, white);
```

Comment:
Since these variables all pertain to the actual form input (and not the field at large - I consider a field to be label + inputs), I'd prefer us to prefix these with ``$input-*`` and leave any ``$field-*`` variables for rules around field combo properties.